app/code/Magento/Sales/Block/Order/Totals.php

```diff
@@ -6,7 +6,7 @@
 namespace Magento\Sales\Block\Order;
 
 use Magento\Sales\Model\Order;
-
+use Magento\Sales\Model\Order\Invoice;
```

Comment:
Hello @SilinMykola 
Thank you for your contribution;
Please don't forget to keep an empty string after imports

```suggestion
use Magento\Sales\Model\Order\Invoice;

```

Reply:
Hello @andrewbess 
Thanks for your reviewing, I fixed your remark.